modules/admin/proc-enabling-telemetry-using-helm.adoc

```diff
@@ -0,0 +1,44 @@
+[id="proc-enabling-telemetry-using-helm_{context}"]
+= Enabling telemetry using the Helm Chart
+
+You can enable the telemetry data collection feature by using the Helm Chart.
+
+.Procedure
+
+. In the *Developer* perspective of the web console, go to the *Helm* view to see the list of Helm releases.
+. Click the *overflow* menu on the Helm release that you want to use and select *Upgrade*.
++
+[NOTE]
+====
+You can also create a new Helm release by clicking the *Create* button and edit the configuration to enable telemetry.
+====
+
+. Use either the *Form* view or *YAML* view to edit the Helm configuration:
+** Using *Form view*
++
+.. Expand *Root Schema → global → Dynamic plugins configuration. → List of dynamic plugins that should be installed in the backstage application*.
+.. Click the *Add list of dynamic plugins that should be installed in the backstage application.* link.
+.. In the *Package specification of the dynamic plugin to install. It should be usable by the npm pack command*. field, add the following value:
++
+`./dynamic-plugins/dist/janus-idp-backstage-plugin-analytics-provider-segment`
```

Comment:
If the user has first to disable it, that means that if he is enabling it, the value is already there
So he should find the field `Package specification of the dynamic plugin to install. It should be usable by the npm pack command` with the value `./dynamic-plugins/dist/janus-idp-backstage-plugin-analytics-provider-segment`
and clear the corresponding `Disable the plugin` checkbox. If it is missing, he should be adding it (but if it is missing, that means it is already disabled - by default).